aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -104,7 +104,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.code.request_publish.per_byte, "code.request_publish.per_byte", 2 * MUL],
 
     // Note(Gas): These are storage operations so the values should not be multiplied.
-    [.event.write_to_event_store.base, "event.write_to_event_store.base", 500_000],
+    [.event.write_to_event_store.base, "event.write_to_event_store.base", 300_000],
```

Comment:
is this intended?

Reply:
Yes, this is set according to Alden's doc. Although it should be clear that we still haven't finalized those values and they are subject to change